src/pages/Home/index.jsx

```diff
@@ -1,9 +1,38 @@
+import { NotesList } from "components";
+import { useNotes } from "contexts/";
+
 const Home = () => {
-    return (
-        <section className="section-wrapper flex-col flex-align-center flex-justify-center">
-            Home
-        </section>
-    )
-}
-
-export { Home };
\ No newline at end of file
+	const { notes, notesLoading, notesError } = useNotes();
+
+	const loadingMessage = notesLoading && (
+		<div className="message">
+			<p className="success-color text-lg my-1">Loading Notes...</p>
+		</div>
+	);
+	const errorMessage = notesError && (
```

Comment:
Like how you're handling error all over your app through context. I'll also try this. Learned a new way. Great work!

Reply:
Thank you!